sql_metadb/derived_tables/holdings_tags

```diff
@@ -0,0 +1,27 @@
+-- This derived table extracts tags from the holdings record.
+DROP TABLE IF EXISTS holdings_tags;
+
+CREATE TABLE holdings_tags AS 
+SELECT 
+    hd.instance_id,
+    hd.id AS holdings_id,
+    hd.hrid AS holdings_hrid,
+    hldtags.jsonb #>> '{}' AS holdings_tags,
```

Comment:
This column name should probably be holdings_tags. I changed it for the instance tag column name to be singular. This is just to follow that all column names be singular.